test/actions/pulp3/orchestration/copy_all_units_test.rb

```diff
@@ -568,6 +568,28 @@ def test_all_module_streams_copied_if_empty_modular_filter_rules
       assert_equal @repo.module_streams.pluck(:id).sort, @repo_clone.module_streams.pluck(:id).sort
     end
 
+    def test_module_streams_copied_from_actual_source_repo
+      # Try to copy a module stream from a source repository that doesn't have it.
+      filter = FactoryBot.build(:katello_content_view_module_stream_filter, :inclusion => true)
+      duck = @repo.module_streams.where(:name => "duck").first
+      FactoryBot.create(:katello_content_view_module_stream_filter_rule,
+                                   :filter => filter,
+                                   :module_stream => duck)
+      @repo.root.update!(:url => 'https://jlsherrill.fedorapeople.org/fake-repos/needed-errata/')
```

Comment:
It looks like we're using this URL in a few places in Katello. I'm not too sure about referencing external URLs in our dev pipeline just in case this ever goes down. Since there is precedent for using `needed-errata`, I'm ok with it; I think we should at least consider doing away with this in the future.

Reply:
we do this in QE as well, might be something I can bring up in a future JPL call and see what everyone else thinks.